docs/14_extending-kedro.md

```diff
@@ -1,135 +1,44 @@
-# Customise Kedro CLI
+# Adding a custom CLI command
 
-While Kedro provides a number of pre-defined CLI options off the shelf in `kedro_cli.py`,
-you can easily add a custom option in your project by adding a new `@click.option` decorator to a function in `kedro_cli.py`. You can learn more about the click options in [their documentation](https://click.palletsprojects.com/en/7.x/options/).
-
-## Example of adding a custom CLI command
-
-If you want to add a new Kedro command, simply add a `@cli.command()` and a corresponding function. In `kedro_cli.py`, add the new function as follows:
+If you want to add a new custom CLI command, simply create a new `click` group, add a `@cli.command()` to that group and then register the new group with `cli` group defined in `src/<package_name>/cli.py` as follows:
 
 ```python
-from kedro.cli import get_project_context
+# src/<package_name>/cli.py
+...
 
-@cli.command()
+@click.group()
+def custom():
+    """Custom commands."""
+
+@custom.command()
 def to_json():
     """ Display the pipeline in JSON format """
-    context = get_project_context()
-    print(context.pipeline.to_json())
+    package_name = str(Path(__file__).resolve().parent.name)
+    with KedroSession.create(package_name) as session:
+        context = session.load_context()
+        print(context.pipeline.to_json())
+
+...
+cli.add_command(jupyter_group)
```

Comment:
What is the `jupyter_group` here? Is that obvious to others? I'm not seeing that mentioned elsewhere --- can you remove it to avoid confusion?

Reply:
Since users need to add some code to the existing file I tried to show (with `...`) that some existing code is omitted and `cli.add_command(jupyter_group)` is a part of that code after which users need to add their code.